packages/viewer/src/components/scene-tree/lib/controller.ts

```diff
@@ -194,6 +196,38 @@ export class SceneTreeController {
     );
   }
 
+  /**
+   * Invokes a network request that will return the index of the node with the
+   * given ID. If the node is collapsed, all parents will be expanded.
+   *
+   * @param id An ID of an item.
+   * @returns A promise that resolves with the index of the node.
+   */
+  public async locateNode(id: string): Promise<number> {
```

Comment:
The documentation here is good, but I wonder if this should be called `locateNodeIndex`? I could see a case where `locateNode` would maybe be used for returning more information about the specific item.